regression/contracts/assigns_enforce_arrays_05/test.desc

```diff
@@ -1,6 +1,6 @@
 CORE
 main.c
---enforce-contract assigns_ptr --enforce-contract assigns_range
+--enforce-contract assigns_range
```

Comment:
Doesn't quite match the commit message?

Reply:
This was not caught before (not sure why) but I had to fix it for the build to go through. Should I keep it in a separate commit just by itself?

Reply:
I am less worried about where it is but some kind of comment about why you are removing `--enforce-contract assigns_ptr` would be good because I have been trying to figure it out.

Reply:
This should be implementation as a check in the function contracts code -- basically CBMC should only enforce a single function at a time. We could enforce multiple functions, but we would have to havoc the global state and enforce each function under a "clean" (nondet) environment.